openapi.yml

```diff
@@ -3343,6 +3343,10 @@ paths:
         name: owner__isempty
         schema:
           type: boolean
+      - in: query
+        name: query
+        schema:
+          type: string
```

Comment:
I am not sure if it could be of any help but it reminded me that in SFM2 there was quite a nice help for the **djangoql** advanced search https://gitlab.cee.redhat.com/prodsec-dev/sfm2/-/blob/master/sfm2/static/templates/advanced-search-help.html

Reply:
I added a small description